### DISCUSSION TITLE
1 configuration for 1 computer/smartphone

### DISCUSSION THREAD

**Discussion by maldovanov** [wg-easy/wg-easy#663] [Q&A] [answered]
скажите, возможно ли сделать 1 конфигурация для 1 компьютера/смартфона?

я создал пользователя, загрузил конфигурацию в телефон и на компьютер, она работает и там и там, её можно использовать на неограниченное кол-во машин я так понимаю, а хотелось бы чтобы созданный профиль был для 1 человека/компьютера/смартфона

Tell me, is it possible to make 1 configuration for 1 computer/smartphone?

I created a user, loaded the configuration into the phone and onto the computer, it works both ways, it can be used on an unlimited number of machines, as I understand it, but I would like the profile created to be for 1 person/computer/smartphone

**Comment by WeeJeWel** [comment, marked as answer]
Create another user :)

**Reply by eandersons** [reply]
I think what @maldovanov wants is to restrict to use one configuration/user/profile only on one device, but as far as I know it is not possible with WireGuard within container.

**Reply by maldovanov** [reply]
@eandersons you are absolutely right, I was hoping that this would still be possible
